cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/table/field/Fields.java

```diff
@@ -23,10 +23,38 @@
   /**
    * @return the FieldType of INTEGER with the given name.
    */
-  public static FieldType integer(String name) {
+  public static FieldType intType(String name) {
     return new FieldType(name, FieldType.Type.INTEGER);
   }
 
+  /**
+   * @return the FieldType of LONG with the given name.
+   */
+  public static FieldType longType(String name) {
```

Comment:
why the Type in the method name? The class already is named type

Reply:
`long` is a reserved word, hence I've named the method `longType`. Any other suggestions for the name?

Reply:
got it